pkg/apiserver/diagnose/assets_dev.go

```diff
@@ -0,0 +1,10 @@
+//+build dev
+
+package diagnose
+
+import (
+	"net/http"
+)
+
+// var Vfs http.FileSystem = http.Dir("ui/packages/diagnosis_report/build")
+var Vfs http.FileSystem = http.Dir("/Users/baurine/Codes/Work/pingcap-incubator/tidb-dashboard/ui/packages/diagnosis_report/build")
```

Comment:
To fix, relative path can't work when run `go generate pkg/apiserver/diagnose/assets_embed.go`, it reports can't find "ui/packages/diagnosis_report/build".

Reply:
fixed it, get the absolute path by os.Getwd() method